src/main/java/orlov/programming/springcoregym/service/CRUDService.java

```diff
@@ -1,5 +1,5 @@
 package orlov.programming.springcoregym.service;
 
-public interface CRUDService<T> extends UpdatableService<T>{
-    void delete(T t);
+public interface CRUDService<E, ID> extends UpdatableService<E, ID>{
```

Comment:
Logic in services is not so straightforward, so no need to use such Interfaces as UpdableService,...
Better to create separate contracts for each service.